specification/sdk-tracing.md

```diff
@@ -260,6 +260,53 @@ high contention in a very high traffic service.
 * `maxExportBatchSize` - the maximum batch size of every export. It must be
   smaller or equal to `maxQueueSize`. The default value is `512`.
 
+##### Span watcher processor
+
+This implementation passes spans that have been started but not yet ended
+to the configured exporter. These spans are called _unfinished_ spans.
+On each interval defined by `reportIntervalMillis`, all unfinished spans that
+have been started more than `reportIntervalMillis` ago, are exported.
+
+**Configurable parameters:**
+
+* `exporter` - the exporter where the spans are pushed.
+* `maxWatchedSpans` - the maximum number of unfinished spans to be watched.
+  While reached, no new spans will be accepted. The default value is `2048`.
+* `reportIntervalMillis` - the delay interval in milliseconds between two
+  consecutive exports. The default value is `5000`.
+* `exporterTimeoutMillis` - how long the export can run before it is cancelled.
+  The default value is `30000`.
+* `maxExportBatchSize` - the maximum batch size of every export. It must be
+  smaller or equal to `maxWatchedSpans`. The default value is `512`.
+* `exportEndedSpans` (boolean) - if set, spans are also exported once they are
+  ended.
+  The default value is `false`.
+* `spanDurationTimeoutMillis` - On each report interval, spans older than
+  `spanDurationTimeoutMillis` which have not yet ended will be dropped from
+  the watch list.
+  The default is `60000`.
+  (_only applies to languages where weak references are not supported or
+  suitable_)
+
+In order to avoid multiple exports potentially being blocked on connection issues or
+when `reportIntervalMillis` is set inappropriately, exports should *not* be performed
+in parallel. If a previous export is still in progress on the next
+`reportIntervalMillis`, the current export should be skipped.
+
+For languages that support the concept of weak references it is recommended to
+only use these for keeping references to unfinished spans in the processor.
+Otherwise, it would accumulate references to spans abandoned by the user without
+ending them and therefore potentially leak memory.
+If weak references are not supported or suitable to be used, the timeout parameter
+`spanDurationTimeoutMillis` should be added for this processor. Unfinished spans which
+were first provided to the processor longer than this timeout ago are dropped from the
+list of watched spans and therefore updates will no longer be reported for them.
+This timeout, however, should only apply to this processor and MUST NOT affect spans
+that are ended properly after the timeout. If `exportEndedSpans` is set, the span MUST
+still be exported once ended. Other processors, including the simple and batching
+processor, will still export these spans as they normally would.
```

Comment:
Consider to remove `Other processors, includi...` as it should be anyway clear that one processor should not impact others in any aspect.